.github/workflows/ci.yml

```
@@ -44,6 +44,9 @@ jobs:
           - python-version: 3.9
             omp: n
             mpi: n
+          - python-version: "3.11"
```

```suggestion
          - python-version: 3.11
```
I think the quotes are needed for python 3.10 as that becomes 3.1 but for 3.11 I think it will be fine without the quotes just like 3.8 and 3.9 are

---

If anything, this is an argument for putting everything in quotes --- leaving out quotes means that it will get parsed as a float in YAML, which happens to work for everything other than 3.10 but doesn't seem like what you'd want for parsing a version number in general.

---

Just updated this to use quotes consistently

---

Assuming CI passes, do you want to go ahead and change the majority of our testing over to Python 3.11?

---

That's what I was originally going to do, but then I opted to just leave everything as it is currently. I may switch over to have 3.11 default once 3.7 reaches EOL and we drop it (June 27).